docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -754,6 +1245,32 @@ If you want to have a customized UI, here are some hints:
 
    ![](/docu/media/10.1_CustomApplicationUI.png)
 
+## Relevant functions and events
+
+There are some functions and events which are quite relevant for the CSK approach and you should know about to understand the background of the code.
+
+Some were already explained within the other chapters. Following you will find some further explainations.
+
+Mostly all these functions and events will exist for each module per default.
+
+### Function: resetModule
+
+Each module should provide a function to reset itself. This function should stop all processing of the module which are not needed per default.
+
+This function should register to the "CSK_PersistentData.OnResetAllModules" event.
+
+This is very helpful if you run multiple processes in different modules but you want to switch to another application and you don't want that unused processes run in the background.
+
+By notifying the "CSK_PersistentData.OnResetAllModules" event all modules will automatically stop and will wait for a new setup.
+	  
+### Function: getStatusModuleActive
+
+As CSK modules are generic, it is possible that you can run CSK based applications on different devices.
+
+Some modules and their features might be not usable on specific devices (due to missing APIs) but still can be installed to the device without errors.
+
+Each module should have a function named "getStatusModuleActive" to give feedback about the status if the module can be used on the current used device so that applications can react on this information.
```

Comment:
```suggestion
Each module should have a function named "getStatusModuleActive" to give feedback about the status if the module can be used on the currently used device so that applications can react on this information.
```